[2024-12-17T04:43:04.330+0000] {processor.py:186} INFO - Started process (PID=434278) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:43:04.331+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:43:04.334+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:43:04.334+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:43:04.340+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:43:04.339+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:43:04.340+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:43:04.340+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:43:04.341+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:43:04.354+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-12-17T04:45:16.283+0000] {processor.py:186} INFO - Started process (PID=434383) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:45:16.285+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:45:16.287+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:45:16.287+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:45:16.292+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:45:16.291+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:45:16.293+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:45:16.292+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:45:16.293+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:45:16.306+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2024-12-17T04:46:12.348+0000] {processor.py:186} INFO - Started process (PID=434445) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:46:12.350+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:46:12.353+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:46:12.352+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:46:12.357+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:46:12.356+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:46:12.357+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:46:12.357+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:46:12.358+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:46:12.371+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2024-12-17T04:55:06.553+0000] {processor.py:186} INFO - Started process (PID=434605) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:55:06.554+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:55:06.558+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:55:06.557+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:55:06.564+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:55:06.563+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:55:06.564+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:55:06.564+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:55:06.565+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:55:06.584+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-12-17T04:56:04.950+0000] {processor.py:186} INFO - Started process (PID=434667) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:56:04.952+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:56:04.956+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:56:04.955+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:56:04.961+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:56:04.960+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:56:04.962+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:56:04.962+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:56:04.962+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:56:04.982+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2024-12-17T04:57:01.683+0000] {processor.py:186} INFO - Started process (PID=434728) to work on /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:57:01.684+0000] {processor.py:914} INFO - Processing file /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-17T04:57:01.688+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:57:01.687+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:57:01.694+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:57:01.693+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-17T04:57:01.695+0000] {logging_mixin.py:190} INFO - [2024-12-17T04:57:01.694+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-17T04:57:01.695+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-17T04:57:01.715+0000] {processor.py:208} INFO - Processing /root/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
